app/main.py

```diff
@@ -1,6 +1,14 @@
-from typing import Callable
+from typing import Any, Callable
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    funct_results_cache = {}
```

Comment:
```suggestion
    cached_value = {}
```

Reply:
To make the variable name more descriptive, consider using a name that conveys the specific purpose of the cache and the kind of results it stores.